src/transformers/file_utils.py

```diff
@@ -1168,15 +1194,33 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> model = {model_class}.from_pretrained("{checkpoint}")
 
     >>> # audio file is decoded on the fly
-    >>> inputs = feature_extractor(dataset[0]["audio"]["array"], return_tensors="pt")
-    >>> logits = model(**inputs).logits
+    >>> inputs = feature_extractor(dataset[0]["audio"]["array"], sampling_rate=sampling_rate, return_tensors="pt")
+
+    >>> with torch.no_grad():
+    ...     logits = model(**inputs).logits
+
     >>> predicted_class_ids = torch.argmax(logits, dim=-1)
     >>> predicted_label = model.config.id2label[predicted_class_ids]
+    >>> predicted_label
+    {expected_output}
+
```

Comment:
black doesn't seem to be happy with that. Think we need to slightly change `style_doc.py` to leave an empty line between expected output for the doc test no?

Reply:
cc @sgugger

Reply:
I have no idea what you mean here. Black does not touch parts of code between three ".